events/wc_offmap_events.txt

```diff
@@ -961,4 +961,453 @@ character_event = {
 			}
 		}
 	}
+}
+character_event = {
+	id = WCOME.500
```

Comment:
Is there a reason why you created a new event?

Reply:
cause it works better than just decision?

Reply:
also if you mean satanist one i just like mine better

Reply:
I mean, why you not used `MNM.7118`?

Reply:
because mine heals more stuff and adds mutations etc and theres no point in using vanilla if its just completely rewritten

Reply:
Mutations can be added.
Don't get what means "heal more stuff".

Reply:
you can choose what category of things you want to heal and get all bad traits removed

Reply:
It's a bad copypaste. Doesn't add scars, doesn't clear glass eye, keep prosthetic hands/legs, doesn't clear peg lags, doesn't give immunity. Even if you add it, it still will be a copypaste so you either need to adapt the vanilla effect somehow or replace all the copypaste's by scripted effects and triggers.

Reply:
It doesn't clear incapable.

Reply:
thats because its not a copypaste i based it on scripted_triggers i used